### PR TITLE
Display app name as "Rancher Desktop by SUSE"

### LINKED ISSUE #5822: Add SUSE branding within App Windows
### Problem Description

Currently, the only place I see the SUSE name in the App is in the about menu where it calls out the copyright holder.

We would like people to know that Rancher Desktop comes from SUSE. There are people who don't put the name connection between Rancher and SUSE.

### Proposed Solution

Put the name SUSE in the main window screen. Some ideas (but not prescriptions):

- The name being "Rancher Desktop by SUSE"
- By SUSE in the footer
- More from SUSE in the left navigation with some links back to other things we have

These are just suggestions to seed any brainstorming.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by jandubois**
Both in the "General" page and the "About" box.

Fixes #5822

### CODE REVIEW

File: pkg/rancher-desktop/main/mainmenu.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ function getHelpMenu(isMac: boolean): MenuItem {
         role:  'about',
         label: `&About ${ Electron.app.name }`,
         click() {
+          Electron.app.setAboutPanelOptions({"applicationName": "Rancher Desktop by SUSE"});
```
Comment (rak-phillip) [inline pkg/rancher-desktop/main/mainmenu.ts:60]: Instead of setting this here, let's update `setAboutPanelOptions` in `background.ts`

**Review by rak-phillip — APPROVED** [review]
LGTM

How do we feel about the line wrapping in the first run dialog?

image

**Comment by rak-phillip** [comment]
@jandubois I updated this PR so that we increase the width of the first run dialog to accommodate the updated title.

**Comment by jandubois** [comment]
Looks good now:

CleanShot 2023-10-24 at 17 21 37@2x

**Review by rak-phillip — APPROVED** [review]

**Pull request merged**